plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -78,4 +80,114 @@ public void testTableWithRepeatedAndMissingColumnNames()
         assertQuery("desc table_with_duplicate_and_missing_column_names", "SELECT * FROM (VALUES('a','varchar','','')," +
                 " ('column_1','varchar','',''), ('column_2','varchar','',''), ('c','varchar','',''))");
     }
+
+    // test polymorphic table function
```

Comment:
we don't use block comments like this. they always flow away from a block of code they are decorating (or the block of code gains new members mixed in)

Reply:
Removed.